examples/otlp-exporter-node/docker/docker-compose.yaml

```diff
@@ -10,7 +10,7 @@ services:
     ports:
       - "9464:9464"
       - "4317:4317"
-      - "55681:55681"
+      - "4318:4318"
```

Comment:
same, collector 0.25.0 does not expose 4318 as default. This change might break the example.